lib/timescaledb/rails/hyperfunctions.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Timescaledb
+  module Rails
+    # :nodoc:
+    module Hyperfunctions
+      TIME_BUCKET_ALIAS = 'time_bucket'
+
+      # @return [Array<ActiveRecord::Base>]
+      def time_bucket(interval, target_column = nil)
+        target_column ||= hypertable.time_column_name
```

Comment:
Use the new `hypertable_time_column_name` as it memoizes the value to prevent extra queries to the database.